src/backend/distributed/planner/multi_explain.c

```diff
@@ -245,7 +238,10 @@ CoordinatorInsertSelectExplainScan(CustomScanState *node, List *ancestors,
 	ExplainOpenGroup("Select Query", "Select Query", false, es);
 
 	/* explain the inner SELECT query */
-	ExplainOneQuery(query, 0, into, es, queryString, params, NULL);
+	IntoClause *into = NULL;
+	ParamListInfo params = NULL;
+	char *queryString = NULL;
+	ExplainOneQuery(queryCopy, 0, into, es, queryString, params, NULL);
```

Comment:
https://github.com/citusdata/citus/issues/3947 was because we passed the original query instead of its copy.

Reply:
Optional: could we explain selectPlanForInsertSelect directly now?